### DISCUSSION TITLE
[BUG]assets目录权限

### DISCUSSION THREAD

**Discussion by MasterY121** [1Panel-dev/1Panel#6577] [Q&A]
### 联系方式

1809992402@qq.com

### 1Panel 版本

v1.10.16-lts

### 问题描述

访问localhost:15555/assets/ 可访问到部分资源

### 重现步骤

在目录扫描中我发现assets目录是可访问的，访问其他目录则会返回“暂无权限访问”
![image](https://github.com/user-attachments/assets/37621c0f-745f-4a6e-902c-a5e66b0509a9)
![image](https://github.com/user-attachments/assets/c3b7e132-e01d-4a9b-accd-0e26914ea600)

### 期待的正确结果

返回“暂无权限访问”
![image](https://github.com/user-attachments/assets/c3b7e132-e01d-4a9b-accd-0e26914ea600)

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，目前登录页加载的部分静态资源允许匿名访问。
